src/test/java/org/folio/circulation/domain/LoanCheckInServiceTest.java

```diff
@@ -0,0 +1,136 @@
+package org.folio.circulation.domain;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.Collections;
+import java.util.UUID;
+
+import org.folio.circulation.domain.representations.CheckInByBarcodeRequest;
+import org.joda.time.DateTime;
+import org.junit.Test;
+
+import api.support.builders.CheckInByBarcodeRequestBuilder;
+import api.support.builders.ItemBuilder;
+import api.support.builders.LocationBuilder;
+import io.vertx.core.json.JsonObject;
+
+public class LoanCheckInServiceTest {
```

Comment:
What about the situation when an item is checked in to a service point that does not serve the home location?

Reply:
Added, thanks.